admin_app_2/src/components/LanguageButtonBar.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Layout } from "@/components/Layout";
+import {
+  DEFAULT_LANGUAGE,
+  LANGUAGE_OPTIONS,
+  appColors,
+  appStyles,
+  sizes,
+} from "@/utils";
+import { Menu, MenuItem, ToggleButton, ToggleButtonGroup } from "@mui/material";
+import { AddCircle, Delete, InfoOutlined } from "@mui/icons-material";
+
+const LanguageButtonBar = ({ expandable }: { expandable: boolean }) => {
+  const [langList, setLangList] = React.useState(
+    expandable
+      ? [LANGUAGE_OPTIONS.find((l) => l.code === DEFAULT_LANGUAGE)]
+      : LANGUAGE_OPTIONS
+  );
+  const [selectedLang, setSelectedLang] = React.useState<string>("en");
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  return (
+    <Layout.FlexBox
```

Comment:
This language bar should become scrollable if smaller than the number of tabs. No idea how hard that is. Can make a ticket for it. Right now it pokes out if screen is made too small

Reply:
Yes, let's ticket this